error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/ReactorRulesTestInput.java

```diff
@@ -408,46 +408,50 @@ StepVerifier.FirstStep<Integer> testStepVerifierFromFlux() {
 
   ImmutableSet<StepVerifier.Step<Integer>> testStepVerifierStepIdentity() {
     return ImmutableSet.of(
-        StepVerifier.create(Mono.just(1)).expectNext(),
-        StepVerifier.create(Mono.just(2)).expectNextCount(0L));
+        Mono.just(1).as(StepVerifier::create).expectNext(),
+        Mono.just(2).as(StepVerifier::create).expectNextCount(0L),
+        Mono.just(3).as(StepVerifier::create).expectNextSequence(ImmutableList.of()),
+        Mono.just(4).as(StepVerifier::create).expectNextSequence(ImmutableList.of(5)));
   }
 
   ImmutableSet<StepVerifier.Step<String>> testStepVerifierStepExpectNext() {
     return ImmutableSet.of(
-        StepVerifier.create(Mono.just("foo")).expectNextMatches(s -> s.equals("bar")),
-        StepVerifier.create(Mono.just("baz")).expectNextMatches("qux"::equals));
+        Mono.just("foo").as(StepVerifier::create).expectNextMatches(s -> s.equals("bar")),
+        Mono.just("baz").as(StepVerifier::create).expectNextMatches("qux"::equals));
```

Comment:
These `.as(StepVerifier::create)` changes were not previously flagged by the tests, because the actual rule under test suggested a smaller replacement. The reason I noticed this now, is that for the `StepVerifierStepIdentity` test cases I added a negative entry, in order to validate the `@Matches` usage.